### ISSUE TITLE
Change constant `git_ignore_lines` element `.Rdata` to`.RData`

### ISSUE THREAD

**Issue by dchiu911** [r-lib/usethis#1677]
I believe the standard R workspace is called `.RData`. However, the constant `git_ignore_lines` has the value ".Rdata".  So if the user already has `.RData` in their `.gitignore` and runs `usethis::use_git()` to initialize a repo,  they'd have `.Rdata` added when `.RData` is the same

**Comment by jennybc** [comment]
Yeah from the history it looks like `use_git()` used to ignore `.Rhistory`, `.RData`, and `.Rproj.user` and `git_vaccinate()` ignored `.Rproj.user`, `.Rhistory`, `.Rdata`, `.httr-oauth`, and `.DS_Store` and then they were harmonized (https://github.com/r-lib/usethis/commit/26d2c4460ab9cb605a3d0d6682cfcf1f20c8285b) with `.Rdata` surviving, when it seems like `.RData` would make more sense.

Would you like to make a PR? This should have a NEWS bullet.
